introduction-to-bedrock/java/README.md

```diff
@@ -0,0 +1,24 @@
+# Java Bedrock REST Api Samples
+
+This repository contains sample of how you can call `invokeModel` REST API and sign it with AWSSigV4 for Amazon Bedrock using Java programming language. The example uses Meta LLama 2 model but you can substitute it with any currently available model isd found on [AWS website](https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids-arns.html).
+
+## Contents
+
+- [Run invokeModel API with Java](./my-app/src/main/java/com/mycompany/app/App.java) - Call invokeModel Amazon Bedrock REST API and sing it with AWSSigV4 using Java programming language
+
+## Getting Started
+
+To get started with the samples, follow these steps:
+
+1. Clone the repository: `git clone https://github.com/your-username/your-repo.git`
```

Comment:
Please change the clone command to be the exact repo link for this repo.

Reply:
Updated.